Dockerfile

```diff
@@ -1,12 +1,24 @@
 FROM gitlab-registry.cern.ch/swan/docker-images/dev-jupyter
+
+LABEL maintainer="Akash Ravi <akashkravi@gmail.com>"
```

Comment:
Instead of `LABEL`, you can user `MAINTAINER`

Reply:
I found this method on https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
And now I know :)

Reply:
Thanks